### DISCUSSION TITLE
Contact

### DISCUSSION THREAD

**Discussion by Rockey7164** [sleeyax/aternos-discord-bot#33] [General]
Hey i had come accross some issues for creating the bot for myself is there anyway i can contact you through discord or any other platform you like it would be great if you could hit me up in discord 

@sleeyax

**Comment by sleeyax** [comment]
If it's a general issue, use the search function on GitHub first to check if your issue has already been answered. If not - and your question/issue doesn't contain confidential/private information - I'd prefer if you could [create an issue](https://github.com/sleeyax/aternos-discord-bot/issues/new/choose) so other people who encounter the same kind of issues in the future can benefit from the solution as well.

If that's not possible lmk your discord username so I can add you. Keep in mind that I'm not your personal tech sport guy though lol. Google can be more helpful in most cases.

**Comment by Fishhyyy** [comment]
The commands won’t work and bot is offline
